index.js

```diff
@@ -738,9 +745,49 @@ class BitbucketScm extends Scm {
      * @param  {Response}    Object          Object containing stats for the scm
      */
     stats() {
-        return this.breaker.stats();
+        const scmContexts = this._getScmContexts();
+        const scmContext = scmContexts[0];
+        const stats = this.breaker.stats();
+
+        return { [scmContext]: stats };
+    }
+
+   /**
+    * Get an array of scm context (e.g. bitbucket:bitbucket.org)
+    * @method getScmContexts
+    * @return {Array}
+    */
+    _getScmContexts() {
+        // TODO: return fixed value temporarily.
+        // need to change if the other bitbucket scm is supported.
+        const contextName = ['bitbucket:bitbucket.org'];
+
+        return contextName;
     }
 
+   /**
+    * Determin if a scm module can handle the received webhook
+    * @method canHandleWebhook
+    * @param {Object}    headers     The request headers associated with the webhook payload
+    * @param {Object}    payload     The webhook payload received from the SCM service
+    * @return {Promise}
+    * */
+    _canHandleWebhook(headers, payload) {
+        return Promise.all([
+            this._getScmContexts(),
+            this._parseHook(headers, payload)
+        ]).then(([scmContexts, parseResult]) => {
+            const scmContext = scmContexts[0];
+            const scmHostName = scmContext.split(':')[1];
```

Comment:
I think we shouldn't parse the scmContext value because it doesn't have a spec.
You can define a variable like `hostname`, with fixed value temporarily, and use it for generating scmContext and comparing against checkoutUrl.